Dockerfile.tmpl

```diff
@@ -107,24 +108,15 @@ RUN conda config --add channels nvidia && \
     /tmp/clean-layer.sh
 
 # Install spacy
+# b/232247930: uninstall pyarrow to avoid double installation with the GPU specific version.
 {{ if eq .Accelerator "gpu" }}
-RUN mamba install -y -c conda-forge spacy cupy cuda-version=$CUDA_MAJOR_VERSION.$CUDA_MINOR_VERSION && \
+RUN pip uninstall -y pyarrow && \
```

Comment:
Let's also uninstall grpc-cpp (hopefully nothing actually needs it...)

Reply:
d. uninstalled via mamba, pip doesn't seem to find it.